### DISCUSSION TITLE
CTS700 on a Compact P

### DISCUSSION THREAD

**Discussion by leonkdk** [veista/nilan#118] [Q&A] [answered]
Hi Veista.

I am trying to add the Compact P to my Home Assistant. I have a CTS700 interface and as such I hope you will be able to assist me in the setup process ;-)

Best regards

**Comment by veista** [comment]
Hi,

Sorry I can't help with that.

**Reply by lauer** [reply]
Its it so much different than the CTS602? It seems to be the same modbus interfaces?

**Comment by flemmingtoo** [comment, marked as answer]
> Hi Veista.
> 
> I am trying to add the Compact P to my Home Assistant. I have a CTS700 interface and as such I hope you will be able to assist me in the setup process ;-)
> 
> Best regards

Try this
https://gist.github.com/jakobmoll/d1742736a4027c59e6c1c053fbb27c80
